localization/ar_tag_based_localizer/package.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ar_tag_based_localizer</name>
+  <version>0.0.0</version>
+  <description>The ar_tag_based_localizer package</description>
+  <maintainer email="shintaro.sakoda@tier4.jp">Shintaro Sakoda</maintainer>
+  <maintainer email="masahiro.sakamoto@tier4.jp">Masahiro Sakamoto</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>autoware_cmake</buildtool_depend>
+
+  <depend>aruco</depend>
+  <depend>autoware_auto_mapping_msgs</depend>
+  <depend>cv_bridge</depend>
+  <depend>geometry_msgs</depend>
+  <depend>image_transport</depend>
+  <depend>lanelet2_extension</depend>
+  <depend>rclcpp</depend>
+  <depend>sensor_msgs</depend>
+  <depend>tf2_eigen</depend>
```

Comment:
Is this necessary? (commenting out here and `#include <tf2_eigen/tf2_eigen.hpp>` in `tf_tag_caster_core.cpp`, and still the build passes, so apparently not necessary?)

Reply:
Sorry my mistake